### ISSUE TITLE
Error on Raygun.detach()

### ISSUE THREAD

**Issue by gregor** [MindscapeHQ/raygun4js#302]
In the latest 2.13.0 a call to `Raygun.detach()` will result in an error. It fails to detach the `PromiseRejectionHandler` as `detachPromiseRejectionFunction` in line 2727 of rayun.vanilla.js is not defined.

````
  // Uninstall global promise rejection handler.
  function detachPromiseRejectionHandler() {
    detachPromiseRejectionFunction();
  }
````

**Comment by BenjaminHarding** [comment]
Thanks @gregor, we'll get a fix for this in the next version of Raygun4JS

**Comment by gregor** [comment]
Noticed that this actually happens if you erroneously call `detach()` without ever calling `attach()` which could happen due to an error in our codebase. Thanks for making sure there won't be a JS error in the future in this scenario.
